cottoncandy/interfaces.py

```diff
@@ -410,7 +416,8 @@ def download_json(self, object_name):
         json_data : dict
             Dictionary representation of JSON file
         """
-        assert self.exists_object(object_name)
+        if not self.exists_object(object_name):
```

Comment:
Actually, I think I'll change it to just use FileNotFoundError. There's no reason for there to be a difference in the two backends for what I'm doing right now.